### PR TITLE
add favorites analytics

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes APP-1922 & APP-1923

## What changed (plus any additional context for devs)

added analytics events on the dapp browser for adding to favorites and clicking favorite card

### CODE REVIEW

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/dapp-favorites-analytics/8c6f4b9d4174377ba680587b26f78cd745ec0d9c.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/dapp-favorites-analytics/8c6f4b9d4174377ba680587b26f78cd745ec0d9c.ipa) for 8c6f4b9d4174377ba680587b26f78cd745ec0d9c

File: src/analytics/event.ts [hunk 1]
```diff
@@ -570,6 +572,16 @@ export type EventProperties = {
...
+  [event.browserAddFavorite]: {
+    url: string;
+    name: string;
+    image: string;
+  };
+  [event.browserTapFavorite]: {
+    url: string;
+    name: string;
+    image: string;
+  };
```
Comment (walmat) [inline src/analytics/event.ts:584]: i think you can just set these types to `FavoritedSite`

**Review by walmat — APPROVED** [review]
small nitpick but lgtm

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/dapp-favorites-analytics/35939f85b84c4ae2867285abdab7edab4aa2c756.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/dapp-favorites-analytics/35939f85b84c4ae2867285abdab7edab4aa2c756.ipa) for 35939f85b84c4ae2867285abdab7edab4aa2c756

**Review by brunobar79 — APPROVED** [review]
LGTM!

**Pull request merged**